tracker.js

```diff
@@ -35,6 +36,33 @@ function track() {
         if (vcapApplication.application_uris) {
             event.application_uris = vcapApplication.application_uris;
         }
+        if (process.env.VCAP_SERVICES) {
+            // refer to http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES
+            vcapServices =  JSON.parse(process.env.VCAP_SERVICES);
+            if(Object.keys(vcapServices).length > 0) {
+                event.bound_vcap_services = {};
+                // for each bound service count the number of instances and identify used plans
+                Object.keys(vcapServices).forEach(function(service_label) {
+                    event.bound_vcap_services[service_label] = { 
+                                                                 'count': vcapServices[service_label].length,   // number of service_label instances 
+                                                                 'plans': []                                    // (optional) plan information for service_label
+                                                               };
+                    vcapServices[service_label].forEach(function (serviceInstance) {
+                        if(serviceInstance.hasOwnProperty('plan')) {
+                            if(event.bound_vcap_services[service_label].plans.indexOf(serviceInstance.plan) === -1) {
+                                event.bound_vcap_services[service_label].plans.push(serviceInstance.plan);
```

Comment:
Changed the behavior as requested. Output for the following scenario:
*    two instances of Cloudant (both instances use the _Shared_ plan)
*    one instance of RedisCloud (_30mb_ plan)
*    one user-provided service instance (without plan association)
*    two Object-Storage instances (one each of _standard_ plan and _Free_ plan)

```
  "runtime": "nodejs",
  "bound_vcap_services": {
    "cloudantNoSQLDB": {
      "count": 2,
      "plans": [
        "Shared",
        "Shared"
      ]
    },
    "rediscloud": {
      "count": 1,
      "plans": [
        "30mb"
      ]
    },
    "user-provided": {
      "count": 1
    },
    "Object-Storage": {
      "count": 2,
      "plans": [
        "standard",
        "Free"
      ]
    }
  }
```